packages/react/src/primitives/shared/responsive/utils.ts

```diff
@@ -3,10 +3,10 @@ import { isDesignToken } from '@aws-amplify/ui';
 import { Breakpoint, Breakpoints } from '../../types/responsive';
 
 export const getValueAtCurrentBreakpoint = (
-  values: Record<string, any> | string[] | string,
+  values: Record<string, string> | string[] | string,
   breakpoint: Breakpoint,
   breakpoints: Breakpoints
-) => {
+): string | null => {
```

Comment:
Will this mean I can't do this anymore?
```
const showSomething = useBreakpointValue({
    base: false,
    medium: true,
  });

//...

{showSomething ? <Something /> : null}
```

Reply:
Yeah on second thoughts, the value should allow to be anything when using this hook.